apps/arjuna_map/index.html

```diff
@@ -0,0 +1,523 @@
+                    "43.299428",
+                    "-76.217933",
+                    "New York"
+                ],
+                [
+                    "OH",
+                    "40.417287",
+                    "-82.907123",
+                    "Ohio"
+                ],
+                [
+                    "OK",
+                    "36.007752",
+                    "-97.092877",
+                    "Oklahoma"
+                ],
+                [
+                    "OR",
+                    "43.804133",
+                    "-120.554201",
+                    "Oregon"
+                ],
+                [
+                    "PA",
+                    "41.203322",
+                    "-77.194525",
+                    "Pennsylvania"
+                ],
+                [
+                    "PR",
+                    "18.220833",
+                    "-66.590149",
+                    "Puerto Rico"
+                ],
+                [
+                    "RI",
+                    "40.803221",
+                    "-69.087749",
+                    "Rhode Island"
+                ],
+                [
+                    "SC",
+                    "33.836081",
+                    "-81.163725",
+                    "South Carolina"
+                ],
+                [
+                    "SD",
+                    "44.769515",
+                    "-100.901813",
+                    "South Dakota"
+                ],
+                [
+                    "TN",
+                    "36.117491",
+                    "-86.580447",
+                    "Tennessee"
+                ],
+                [
+                    "TX",
+                    "31.968599",
+                    "-98.901813",
+                    "Texas"
+                ],
+                [
+                    "UT",
+                    "39.32098",
+                    "-112.09373",
+                    "Utah"
+                ],
+                [
+                    "VA",
+                    "37.431573",
+                    "-78.656894",
+                    "Virginia"
+                ],
+                [
+                    "VT",
+                    "44.558803",
+                    "-72.577841",
+                    "Vermont"
+                ],
+                [
+                    "WA",
+                    "47.751074",
+                    "-120.74013",
+                    "Washington"
+                ],
+                [
+                    "WI",
+                    "44.78444",
+                    "-89.787868",
+                    "Wisconsin"
+                ],
+                [
+                    "WV",
+                    "38.597626",
+                    "-80.454903",
+                    "West Virginia"
+                ],
+                [
+                    "WY",
+                    "43.475968",
+                    "-108.09028",
+                    "Wyoming"
+                ]
+            ]
+        }
```

Comment:
Could this become an option of the US States Choropleth?

Reply:
Sure, I'll create an issue for this.